src/js/letters/containers/VeteranBenefitSummaryLetter.jsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import _ from 'lodash';
+
+import { veteranBenefitSummaryOptionText } from '../utils/helpers';
+import { updateBenefitSummaryOptionsStatus } from '../actions/letters';
+
+class VeteranBenefitSummaryLetter extends React.Component {
+  constructor() {
+    super();
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange(domEvent) {
+    this.props.updateBenefitSummaryOptionsStatus(domEvent.target.id, domEvent.target.checked);
+  }
+
+  render() {
+    const benefitInfo = this.props.benefitSummaryOptions.benefitInfo;
+    const optionsToInclude = this.props.optionsToInclude;
+    let militaryServiceInfo;
+    let vaBenefitInfoRows = [];
+
+    _.forIn(benefitInfo, (value, key) => {
+      const optionText = veteranBenefitSummaryOptionText(benefitInfo, key);
+
+      // There are 2 conditions this is checking for
+      // 1. If the current benefit is not in the option text list from helpers,
+      // it means this benefit is not for veterans, and therefore it shouldn't
+      // be displayed.
+      // 2. If the value of the current benefit is false, we don't want to
+      // display it.
+      if (optionText && value !== false) {
```

Comment:
Actually I want to use not false because sometimes the value could be something else other than true or false. For example, `awardEffectiveDate`'s value is a date, and `monthlyAwardAmount` is a number.